archive_converter.py

```diff
@@ -13,8 +13,67 @@
 # limitations under the License.
 
 import email
+import os
+from typing import List, Dict
 from setup_gmail import Message
 
+class ArchiveMessageIndex(object):
+    def __init__(self):
+        #Maps message subject (str) to starting message of email thread
```

Comment:
```suggestion
        # Maps message subject (str) to starting message of email thread
```

Reply:
Note: while you're doing this, can you delete the trailing whitespace added in this file?

There are a number of instances in this commit.
E.g. the line right below this.

```sh
$ egrep '\s+$' *.py -c
archive_converter.py:11
archive_converter_test.py:3
gerrit.py:0
git.py:0
patch_parser.py:0
patch_parser_test.py:7
setup_gmail.py:0
```

Can do so by opening up a shell and running
```
$ ls *.py | xargs sed -i -E 's/\s+$//'
```

(there are a few instances from the previous PR which will get changed too, but I'm fine with that)